src/System.Security.Cryptography.Cng/ref/System.Security.Cryptography.Cng.cs

```diff
@@ -233,14 +254,20 @@ public enum CngUIProtectionLevels
     }
     public sealed partial class ECDsaCng : System.Security.Cryptography.ECDsa
     {
-        public ECDsaCng() {}
-        public ECDsaCng(int keySize) {}
-        public ECDsaCng(CngKey key) {}
+        public ECDsaCng() { }
+        public ECDsaCng(int keySize) { }
+        public ECDsaCng(System.Security.Cryptography.CngKey key) { }
+        public ECDsaCng(System.Security.Cryptography.ECCurve curve) { }
```

Comment:
There is not currently an ECDsaCng(ECParameters) ctor that matches the static Create(ECParameters) -- is this a problem to add later (still for rtm)? Assuming we want to add the matching ctor